models/staging/tpdm/stage/_tpdm__stage.yml

```diff
@@ -115,10 +115,6 @@ models:
     config:
       tags: ['tpdm']
 
-  - name: stg_tpdm__educator_preparation_programs
-    config:
-      tags: ['tpdm']
-
```

Comment:
duplicate entry in the file, I guess it happened during merge conflict resolving